<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1557</width>
    <height>1049</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="setActiveInstrumentsButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>620</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Active instruments</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="instGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>421</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string>Instrumentation</string>
    </property>
    <widget class="QPushButton" name="instrumentUpdateButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Intruments</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="instrumentTableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>401</width>
       <height>381</height>
      </rect>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>441</x>
      <y>161</y>
      <width>1091</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="wfmGenGroupBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Waveform Generator</string>
       </property>
       <widget class="QLabel" name="waveFormGenLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>271</width>
          <height>16</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QPushButton" name="wvfOutPushButton">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>230</y>
          <width>143</width>
          <height>24</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">bacground: rgb(255, 0, 4);
font: 700 9pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Output OFF</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Icons/buttonOFF.png</normaloff>
          <normalon>Icons/buttonOn.png</normalon>Icons/buttonOFF.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>110</y>
          <width>336</width>
          <height>108</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="wvfFuncLabel">
           <property name="text">
            <string>Function</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="wvfFuncReadLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="wvfFuncComboBox">
           <property name="editable">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>SINE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SQUARE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COS</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="wvfAmpLabel">
           <property name="text">
            <string>Amplitude</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="wvfFreqReadLineEdit_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="wvfAmpLineEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="wvfFreqLabel">
           <property name="text">
            <string>Frequency</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="wvfOffsetReadLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="wvfFreqLineEdit"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="wvfOffsetLabel">
           <property name="text">
            <string>Offset</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="wvfAmpReadLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="wvfOffsetLineEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="oscillGroupBox">
       <property name="title">
        <string>Oscilloscope</string>
       </property>
       <widget class="QLabel" name="oscillLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>271</width>
          <height>16</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="lockInGroupBox">
       <property name="title">
        <string>Lock In</string>
       </property>
       <widget class="QLabel" name="lockInLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>271</width>
          <height>16</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="currSourGroupBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Current Source</string>
       </property>
       <widget class="QLabel" name="currSourLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>271</width>
          <height>16</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QPushButton" name="wvfOutPushButton_2">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>140</y>
          <width>143</width>
          <height>24</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">bacground: rgb(255, 0, 4);
font: 700 9pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Output OFF</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Icons/buttonOFF.png</normaloff>
          <normalon>Icons/buttonOn.png</normalon>Icons/buttonOFF.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>140</y>
          <width>197</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="1">
          <widget class="QLineEdit" name="currSourAmpLineEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="currSourFreqLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="currSourFreqLabel">
           <property name="text">
            <string>Frequency</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="currSourAmpLabel">
           <property name="text">
            <string>Amplitude</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="currSourOffsetLabel">
           <property name="text">
            <string>Offset</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="currSourOffsetLineEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>instGroupBox</zorder>
   <zorder>setActiveInstrumentsButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1557</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
